app/queries/control_rate_query_v2.rb

```diff
@@ -0,0 +1,27 @@
+class ControlRateQueryV2
+  def controlled(region)
+    ReportingPipeline::PatientStatesPerMonth
+      .where(hypertension: "yes", htn_care_state: "under_care")
+      .where(patient_assigned_facility_id: region.facilities)
+      .where("months_since_registration >= ?", Reports::REGISTRATION_BUFFER_IN_MONTHS)
+      .where(htn_treatment_outcome_in_last_3_months: :controlled)
+  end
+
+  def controlled_counts(region, range: nil)
```

Comment:
Should range default to nil? It seems like a required parameter. It looks like it would error on `range.begin` if it were nil.

Reply:
just fixed that up